src/lazy/encoder/binary/v1_1/value_writer.rs

```diff
@@ -741,7 +739,7 @@ macro_rules! annotate_and_delegate_1_1 {
             let value_writer = $crate::lazy::encoder::binary::v1_1::value_writer::BinaryValueWriter_1_1::new(
                 self.allocator,
                 self.buffer,
-                self.config(),
```

Comment:
🗺️ Previously, `fn config(&self) -> ValueWriterConfig` was a required method on `ValueWriter` and `MakeValueWriter`, but I realized it wasn't used in many places.

This PR moves the requirement to the `StructWriter` trait, which only has ~6 impls.